onset_maproom/cess.py

```diff
@@ -0,0 +1,129 @@
+#temporary file for cessation date functions; 
+#these will be moved to the main calc.py file when finished
+
+import calc
+import numpy as np
+import pandas as pd
+import xarray as xr
+
+#params for cess as found in maproom
+search_start_day = 1
+search_start_month = 9
+search_days = 90
+dry_thresh = 5
+min_dry_days = 3
+time_coord = "T"
+et = 5
+taw = 60
+sminit = 0 
+
+rr_mrg = calc.read_zarr_data("/data/aaron/ethiopia-rain-rechunked")
+
+#subset of data for testing
+rr_mrgSub = rr_mrg.sel(T=slice('1981-01-01','1983-12-31'))
+daily_rain = rr_mrgSub.isel(X=200,Y=160)
+
+
+#onset date calc used in seasonal cess date func
+#first date after START_DAY(1), START_MONTH(9) in SEARCH_DAYS(90) 
+#days when the soil water balance falls below DRY_THRESH(5)mm 
+#for a period of DRY_SPELL_LENGTH(3) days
+def cess_date(daily_rain, dry_thresh, min_dry_days, et, taw, sminit, time_coord="T"):
+    water_balance = calc.water_balance(daily_rain, et,taw,sminit,"T")
```

Comment:
are you saying

change input name so `cess_date(soil_moisture, etc)` instead of the input var being called 'daily_rain' 
or 
do you actually want me to first calculate `soil_moisture` outside of `cess_date()` and then use that output as the input for `cess_date()`

Reply:
@remicousin could you clarify this for me?

Reply:
I'll try :) basically both. cess_date should be a function of soil moisture rather than rainfall. Users could use whatever to get soil_moisture to feed into it thus yes, the name of the input should change accordingly. And the arguments that belong to water_balance should disappear from cess_date.
We can talk if easier. I am around, even though there are a bunch of meetings this afternoon.

Reply:
I think I understand what you are saying!